src/main/java/com/incognia/api/IncogniaAPI.java

```diff
@@ -186,6 +188,7 @@ public TransactionAssessment registerLogin(RegisterLoginRequest request)
             .installationId(request.getInstallationId())
             .accountId(request.getAccountId())
             .externalId(request.getExternalId())
+            .policyId(request.getPolicyId())
```

Comment:
shouldn't we also add policy id to payment requests?

Reply:
Maybe we should. But I think it shouldn't be on this PR.

Reply:
+1